### ISSUE TITLE
"No Leave Preview hotkey found even though you have the mod" appearing when I don't have World Preview

### ISSUE THREAD

**Issue by pjagada** [Specnr/MultiResetWall#117]
I believe that the leave preview hotkey is a feature of World Preview, and I don't have World Preview (since I'm playing ssg), but I was getting the error of "No Leave Preview hotkey found even though you have the mod, something went wrong trying to find the key." I think [this is the cause](https://github.com/Specnr/MultiResetWall/blob/7c44ec4671984f71888d31b44a26ba7c7f76fa6a/scripts/functions.ahk#L1145), since it believes that that hotkey should exist by checking for atum.

**Comment by Machliah** [comment]
Yes, you're correct. This is a bug and that entire function is horrible, currently undergoing massive rewrites, I think it should be fine if u just click through them but yeah its annoying.
